content/doc/book/pipeline/syntax.adoc

```diff
@@ -248,6 +248,18 @@ need to contain its own `agent` directive. For example: `agent none`
 label:: Execute the Pipeline, or stage, on an agent available in the Jenkins
 environment with the provided label. For example: `agent { label 'my-defined-label' }`
 
+node:: Execute the Pipeline, or stage, on an agent available in the Jenkins
+environment with the provided label, with additional options. For example:
+[source,groovy]
+----
+agent {
+    node {
+        label 'my-defined-label'
+        customWorkspace '/some/other/path'
```

Comment:
I'd really like to have it here at `node` - I'll add a link to the `Common Options` section in a footnote.